pkg/controllers/managementuser/rbac/prtb_handler.go

```diff
@@ -190,6 +194,11 @@ func (m *manager) ownerExists(uid interface{}) (bool, error) {
 	return len(prtbs) > 0, err
 }
 
+func (m *manager) ownerExistsByNsName(nsAndName interface{}) (bool, error) {
```

Comment:
This isn't used anywhere

Reply:
ah I hadn't pushed the above changes that use this yet